docs/hub/enterprise-hub-tokens-management.md

```diff
@@ -0,0 +1,59 @@
+# Tokens Management
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
+</Tip>
+
+Tokens Management allows organization administrators to control access tokens within their organization, ensuring that only authorized users have access to organization resources.
+
+
+## Viewing and Managing Access Tokens
+
+The token listing feature provides a view of all access tokens within your organization. Administrators can:
+
+* Monitor token usage and identify potential security risks
+* Identify inactive or unused tokens that can be revoked
```

Comment:
```suggestion
- Monitor token usage and identify or prevent potential security risks:
  - unauthorized access to private resources ("leak")
  - scopes of access that are too wide
  - improveable token hygienics (tokens that have not been rotated in a long time, etc) 
- Identify inactive or unused tokens that can be revoked
```

maybe list a few potential, concrete risks that make this a very useful feature for companies